packages/preview/jurz/0.1.0/demo.typ

```diff
@@ -0,0 +1,62 @@
+#import "jurz.typ": *
+#import "@preview/chic-hdr:0.4.0": *
+
+#show: chic.with(
+  chic-footer(
+    // left-side: strong(
+    //     link("mailto:admin@chic.hdr", "admin@chic.hdr")
+    // ),
+    center-side: chic-page-number()
+  ),
+  chic-header(
+    // left-side: emph[jurz Demo],
+    // right-side: counter("rz").at(here())
+  ),
+  chic-separator(1pt),
+  // chic-offset(7pt),
+  chic-height(1.5cm)
+)
+#set page(paper: "din-d6", margin: (inside: 2em, outside: 4em))
```

Comment:
Just a minor thing, but maybe you could add a `fill: white` so that the demo images are readable in GitHub dark mode. This is what it currently looks like for me:

![image](https://github.com/typst/packages/assets/7191192/d364e34b-4203-4373-91ec-f4679674ef3a)

Reply:
True, I'll fix that. Thanks!